src/transactions/transactions-discovery.service.test.ts

```
@@ -168,10 +168,44 @@ describe('TransactionsDiscoveryService', () => {
       expect(params.toBlock).toBe('0x3');
-      expect(params.excludeZeroValue).toBe(true);
+      expect(params.excludeZeroValue).toBeUndefined();
     });
   });
 
+  it('discovers and saves supported zero-value ERC1155 initialization transfers', async () => {
+    const initializationTransfer = makeErc1155Transfer('event-1', [
+      { tokenId: '0x211', value: '0x0' }
+    ]);
+    initializationTransfer.from =
+      '0x0000000000000000000000000000000000000000';
```

ESLint

[error] Delete `⏎·····`

(prettier/prettier)